script/build-git

```diff
@@ -5,7 +5,12 @@ DIR="$1"
 case $(uname -s) in
   Darwin)
     brew install curl zlib pcre2 gettext openssl
-    brew link --force zlib pcre2 gettext openssl;;
+    brew reinstall curl zlib pcre2 gettext openssl
+    for i in zlib pcre2 gettext openssl
```

Comment:
Out of interest, why is `curl` not in the `for` list?

Reply:
Because I apparently did not copy and paste correctly.  Thanks for pointing it out.